setup.py

```diff
@@ -1373,12 +1373,20 @@ class db_found(Exception): pass
             # Sun yellow pages. Some systems have the functions in libc.
             if (host_platform not in ['cygwin', 'qnx6'] and
                 find_file('rpcsvc/yp_prot.h', inc_dirs, []) is not None):
-                if (self.compiler.find_library_file(lib_dirs, 'nsl')):
+                if self.compiler.find_library_file(lib_dirs, 'nsl'):
                     libs = ['nsl']
                 else:
                     libs = []
+                    include_dirs = []
+                if self.compiler.find_library_file(lib_dirs, 'tirpc'):
+                    # Sun RPC has been moved from glibc to libtirpc
+                    libs.append('tirpc')
+                    include_dirs = ['/usr/include/tirpc']
```

Comment:
This code lives in the giant detect_modules() function which is 1184 lines long... The function should be refactored, but that's out of the scope of the PR.

Would you mind to rename include_dirs to nic_incs, as done for for other "include_dirs=" parameters? It would avoid bad surprised with name clashes in this giant function.